src/client/components/PasscodeInput.tsx

```diff
@@ -8,6 +8,7 @@ import { remSpace } from '@guardian/source/foundations';
 interface PasscodeInputProps extends TextInputProps {
 	passcode?: string;
 	fieldErrors?: FieldError[];
+	formRef?: React.RefObject<HTMLFormElement | null>;
```

Comment:
Would this ever be undefined?

Reply:
The way it's currently being used, it's always defined, so will refactor that.